gslab_scons/log.py

```diff
@@ -2,27 +2,42 @@
 import sys
 import shutil
 import subprocess
+import gslab_scons.misc as misc
 from datetime import datetime
 from sys import platform
 from misc import is_unix, check_lfs
 
 
-def start_log(log = 'sconstruct.log'):
+def start_log(mode, vers, log = 'sconstruct.log'):
     '''Begins logging a build process'''
     check_lfs()
+    
+    if not (mode in ['develop', 'cache', 'release']):
+        print("Error: %s is not a defined mode" % mode)
+        sys.exit()
+    if mode == 'release' and vers == '':
+        print("Error: Version must be defined in release mode")
+        sys.exit()
+
+    start_message = "\n{0}{0}{0} New build: " + misc.current_time() + " {0}{0}{0}"
+    with open(log, "a") as f:
+        f.write(start_message.format("*"))
+
     if is_unix():
-        sys.stdout = os.popen('tee %s' % log, 'wb')
+        sys.stdout = os.popen('tee -a %s' % log, 'wb')
     elif platform == 'win32':
-        sys.stdout = open(log, 'wb')
+        sys.stdout = open(log, 'ab')
     sys.stderr = sys.stdout 
+    print start_message.format("^")
+
     return None
 
 
-def log_timestamp(start_time, end_time, filename):
+def log_timestamp(start_time, end_time, filename = 'sconstruct.log'):
     '''Adds beginning and ending times to a log file.'''
     with open(filename, mode = 'r+U') as f:
```

Comment:
Check out this wonderful chart! 
![chart](https://cloud.githubusercontent.com/assets/20933669/22811043/57033b6a-eef1-11e6-92d9-ef6ee5399315.png)